platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -140,6 +142,13 @@ public void init() throws Exception {
         return export(Paths.get(exportDirectory));
     }
 
+    @Override
+    public Collection<Describable> getOptionalMigratableInfo() {
+        return dataMigratables.stream()
```

Comment:
Is there a reason you specifically chose a `LinkedList`? 

Also, you're leaking the collection out and not making a deep copy of it so you're allowing callers to modify the contained objects. Just be aware of that.